server.py

```diff
@@ -89,7 +89,7 @@ class AtrributeError:
 
 def get_last_message():
     """Get the latest message"""
-    page_elements = PAGE.query_selector_all("div[class*='request-']")
+    page_elements = PAGE.query_selector_all("div[class*='lg:w-[calc(100%-115px)]']")
```

Comment:
I changed it to `page_elements = PAGE.query_selector_all("div[class*='markdown']")` for the same reason